service-registry/src/main/java/io/servicecomb/serviceregistry/definition/DefinitionConst.java

```diff
@@ -16,10 +16,12 @@
 
 package io.servicecomb.serviceregistry.definition;
 
+import static io.servicecomb.foundation.common.base.ServiceCombConstants.CONFIG_SERVICE;
+
 public interface DefinitionConst {
     String appIdKey = "APPLICATION_ID";
 
-    String serviceDescriptionKey = "service_description";
+    String serviceDescriptionKey = CONFIG_SERVICE;
 
     String nameKey = "name";
```

Comment:
i'd prefer to move them to foundation-common module, but we can't move them all in one go, because doing so may break backward compatibility. the best we can do is to annotate it with @Deprecated. what do you think?